client/grpc/tmservice/querier.go

```diff
@@ -0,0 +1,220 @@
+package tmservice
```

Comment:
can we call this file server or service?

Reply:
Yes, `service` looks better